### PR TITLE
Added validations to post for histories, changed a typo and leftover

### PR DESCRIPTION
**Pull request by AlejandraBC**
primitives to wrappers for users.

I also had to make some changes to HistoryServiceImpl because it was trying to validate that a null was > 0 and it was a blocker for my validations

### CODE REVIEW

**Review by luisgalindoluna — APPROVED** [review]
Well done. Approved

File: src/main/java/com/agilethought/atsceapi/validator/HistoryValidator.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+	}
+
+	private void validateUserId(String user_id) {
+		if (StringUtils.isBlank(user_id)) {
+			throw new BadRequestException(BAD_REQUEST_MESSAGE + "*USER_ID* " + BAD_REQUEST_MESSAGE_USER_ID);
+		}
+	}
+
+	private void validateTotalHours(Integer totalHours) {
+		if (totalHours == null || totalHours < 0) {
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/atsceapi/validator/HistoryValidator.java:52]: `if (totalHours == null || totalHours < 0)` we are repeating a lot of time this validation. Maybe it's a good idea have a util method to generalize ` utilMethod(intVar) -> return intVar== null || intVar < 0;`

**Review by ivanflores15 — APPROVED** [review]
good job, we can create general methods to avoid duplication of code and putting it into ValidaitonUtils class

**Review by EmiliaGuillen-dev — APPROVED** [review]
looks good

**Comment by luisgalindoluna** [comment]
There is one more conflict, probably due to the last merge I imagine, please fix it before merging.

**Pull request merged**